build/index.js

```diff
@@ -70,7 +70,7 @@ var Front = (function () {
                 return _this.httpCall({ method: 'GET', path: 'conversations' }, null, callback);
             },
             update: function (params, callback) {
-                return _this.httpCall({ method: 'PATCH', path: 'conversations/<conversation_id>' }, params, callback);
```

Comment:
Is this change necessary? Looking from the rest of the code, it seems the `<>` will be replaced with the parameter, doesn't it work like that?

Reply:
I have not made that change, this is built javascript file. Somebody who changed this must have forgotten to build, so it appeared as change on my build.

Reply:
I see, anyway it will work both ways, so it's fine with me.

Reply:
Wait wait...

Reply:
Here: https://github.com/balena-io-modules/front-sdk/commit/0f6d452a2e87d23ecf28181238ca767ffb59d014#diff-13b5b151431c7e7a17f273559ed212d5